### ISSUE TITLE
Zooming using subset or datawc does not work for lambert projection

### ISSUE THREAD

**Issue by danlipsa** [UV-CDAT/uvcdat#2106]
See
https://github.com/UV-CDAT/uvcdat-testdata/blob/d57776bca7ffccf9d5844c618a9866654d4883cf/baselines/vcs/test_vcs_boxfill_lambert_subset.png
and
https://github.com/UV-CDAT/uvcdat-testdata/blob/d57776bca7ffccf9d5844c618a9866654d4883cf/baselines/vcs/test_vcs_boxfill_lambert_datawc.png

**Comment by aashish24** [comment]
@danlipsa is this not resolved in this PR: https://github.com/UV-CDAT/uvcdat/pull/2102

**Comment by danlipsa** [comment]
@aashish24 No, this is a new test which shows up blank, not sure why. Strangely this happens for subsetting not only for zooming using datawc.

**Comment by danlipsa** [comment]
@aashish24 So it seems that it is a problem that was in the code before the PR. We can take a look at it after the PR is merged in
